rim-page.html

```diff
@@ -0,0 +1,327 @@
+            }
+            .hero-image {
+                height: 100%;
+                background-image: url(fistbump-over-desk.jpg);
+                background-position: center;
+                background-repeat: no-repeat;
+                background-size: cover;
+                position: relative;
+            }
+            .hero-text {
+                text-align: left;
+                position: absolute;
+                top: 50%;
+                bottom: 50%;                
+            }           
+            .container-2 {            
+                width: 25%;
+            }
+            .container-3 {
+                position: relative;
+                text-align: left;
+                color: white;
+            }
+            .bb-footer {
+                float: left;
+                width: 20%;
+            }
+            .bb-footer li {
+                display: block;
+            }
+            .copyright {
+                display: block;
+                width: 100%;
+                float: right;
+            }    
+        </style>
+        <script>
+            console.log("Hello world!")
+        </script>
+    </head>
+    <body>
+        <div class="container">
+            <div class="row">
+                <div class="column">
+                    <nav>
+                        <ul>
+                            <li><a href="#Logo">Logo</a></li>
+                            <li><a href="#Products">Products</a></li>
+                            <li><a href="#Solutions">Solutions</a></li>
+                            <li><a href="#Industries">Industries</a></li>
+                            <li><a href="#Partners">Partners</a></li>
+                            <li><a href="#Support">Support</a></li>
+                            <li><a href="#Blog">Blog</a></li>
+                            <li><a href="#Search">Search icon</a></li>
+                            <li><a href="#Headset">Headset icon</a></li>
+                        </ul>
+                    </nav>
+                </div>
+        </div>                                                    
+        <div class="container hero-image">
+            <div class="row">
+                <div class="column">
+                    <div class="hero-text">
+                        <h2>Everyone is working at home. Including cyberattackers.</h2>
+                        <p>The recent SolarWinds security attack proves that remote workers and unsecured endpoints leave businesses exposed. Protection is not enough. Our AI-driven security doesn’t react to cyberthreats—it prevents them. If you believe you’ve had a breach, our BlackBerry® Security Services Incident Response experts can help.</p>
+                        <button type="button">CONTACT SALES</button>
+                    </div>
+                </div>
+            </div>               
+        </div>
+        <div class="row">
+            <div class="column">
+                <main>
```

Comment:
This isn't your main content and so this tag is unnecessary.

Reply:
Where would main tag be used on this page?

Reply:
In this case `<main>` would never truly show up on a page like this.
Here is more info:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/main

If you had a "open positions" page, and in the section where you are listing the positions... that would be held within a <main> container.  Or a blog post.. the actual blog post itself, would be held within <main> (which would also carry an <article> tag for the actual article itself).